### PR TITLE
6 implmentation du front du dtails dun event

### PR DESCRIPTION
**Pull request by Omar1H1**
## How to test my changes

- Access react app file
cd react-app
- install the npm files
npm i
- run the servier locally
npm start

it should be looking like this : 

image

## Checklist
- [x] I didn't over-scope my PR
- [x] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I did not include breaking changes
- [x] I made my own code-review before requesting one

### I included unit tests that cover my changes
- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### I added/updated the documentation about my changes
- [ ] 📜 README.md
- [ ] 📕 docs/*.md
- [x] 📓 other
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Review by Jean-Charles-Audebert — APPROVED** [review]
A compléter : En cas de plusieurs viennoiseries choisies, comment s'affichent les résultats?

File: react-app/src/index.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+}
```
[the remark this answers is not in the document]
Comment (Omar1H1) [reply react-app/src/index.css]: No, you're right, i just removed it, thanks for the feedback :) .

**Review by jebourdeau — APPROVED** [review]

**Review by shiipou — APPROVED** [review]

**Review by shiipou — CHANGES_REQUESTED** [review]
Can you add some documentation about your changes

**Review by tcht-marie — APPROVED** [review]

**Comment by jbouhet79** [comment]
Comment tester votre projet ?

**Review by OlivierV79 — APPROVED** [review]

**Review by tcht-marie — APPROVED** [review]

**Review by ThibaultMesmin — APPROVED** [review]

**Review by morganb75 — APPROVED** [review]

**Pull request merged**